industrial_moveit_benchmarking/launch/joint_interpolated_valgrind.launch

```diff
@@ -3,6 +3,7 @@
   <arg unless="$(arg profile)" name="launch_prefix" value="" />
   <arg     if="$(arg profile)" name="launch_prefix" value="valgrind --tool=callgrind" />
 
-  <node name="joint_interpolated_benchmarking_node" pkg="industrial_moveit_benchmarking" type="joint_interpolated_benchmarking_node" launch-prefix="$(arg launch_prefix)"/>
+  <node name="joint_interpolated_benchmarking_node" pkg="industrial_moveit_benchmarking" type="joint_interpolated_benchmarking_node" 
+  	launch-prefix="$(arg launch_prefix)"/>
```

Comment:
Seems like a layout change only?

Reply:
Yes, I momentarily added `output="screen"` and change the layout for better readability but didn't restore it to its original form when I took it out.

Reply:
Could you please revert this change? Let's not introduce non-functional changes in files when we're moving things around (keeps the PR simpler and more on-topic).

Reply:
I reverted this